src/plugins/chromecastPlayer/plugin.js

```diff
@@ -1079,6 +1109,15 @@ class ChromecastPlayer {
     getPlayerState() {
         return this.getPlayerStateInternal() || {};
     }
+
+    getCurrentPlaylistIndex() {
+        // tbd: update to support playlists and not only album with tracks
+        return this.getPlayerStateInternal()?.NowPlayingItem?.IndexNumber;
+    }
+
+    clearQueue(currentTime) { // eslint-disable-line no-unused-vars
```

Comment:
You should be able to just remove the parameter?

```suggestion
    clearQueue() {
```

Reply:
The current implementation of queue management is still rather limited in jellyfin chromecast, so this method is not really important. I intend to extend that in future PR's, and then the method is needed as playbackmanager expects it:
https://github.com/jellyfin/jellyfin-web/blob/4882d9c8cc931e38d422c4766bc0e5d351ca7f6a/src/components/playback/playbackmanager.js#L3968
I already added this method as placeholder for later, and playbackmanager seems to expect a parameter. So I prefer to leave this as is.